spec/factories/employees.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :employee do
+    sequence :email do |n|
+      "test#{n}@test.com"
+    end
```

Comment:
I suggest writing this in one line
```suggestion
    sequence(:email) { |n| "test#{n}@test.com" }

```